### DISCUSSION TITLE
Grouping objects on the board

### DISCUSSION THREAD

**Discussion by jtwaleson** [darklang/dark#4556] [Feedback]
Now all objects are distributed on a 2d grid quite randomly. It would be nice if you can group them together in a rectangle with a name, so you can bring some order to the chaos.

**Comment by sanderjson** [comment]
I agree! I created an issue [here for grouping](https://github.com/darklang/dark/issues/4408) and [here for snap to grid](https://github.com/darklang/dark/issues/4442). 

+1 for organized canvas
